tests/domain/test_domain_winlose.cpp

```diff
@@ -3,14 +3,45 @@
 
 #include "domain/services/WinLosePolicy.hpp"
 #include "domain/core/GameState.hpp"
+#include "domain/entities/actors/Player.hpp"
 
-int main()
+bool testHpLessThanZero()
```

Comment:
Ми користуємося зміїним підходом для назв функцій
```suggestion
bool test_hp_less_than_zero()
```